### ISSUE TITLE
Selector for classes and functions not working

### ISSUE THREAD

**Issue by dasdebashish17** [spyder-ide/spyder#13131]
### Issue Report Checklist

* [ ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [ ] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
Selector for classes and functions doesn't work most of the time. 
However, it seems to work in debug section `spyder --debug-info verbose`

### What steps reproduce the problem?

1. Enable "Selector for classes and functions"
2. Create a class. Define some member functions
3. Check the active class and function in the selector
![image](https://user-images.githubusercontent.com/10602800/85932374-430c6d00-b8e9-11ea-8b9f-00bc0303ed4e.png)

### What is the expected output? What do you see instead?
Expected to see a list of class and its member functions in the selector.
Observed None in both the drop down list.
![image](https://user-images.githubusercontent.com/10602800/85932387-620aff00-b8e9-11ea-8fb8-9d16f4804ff7.png)

However, when Spyder is launched in Debug mode, it works correctly.
![image](https://user-images.githubusercontent.com/10602800/85932421-a6969a80-b8e9-11ea-8db0-fd73ba56aa41.png)

### Paste Traceback/Error Below (if applicable)

```python-traceback

NA

```

## Versions

* Spyder version: 4.1.3
* Python version: 3.7.3 64-bit
* Qt version: 5.9.6
* PyQt version: PyQt5 5.9.2
* Operating System name/version: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.4.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.5.2 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.4 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.0.4 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.17 (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by goanpeca** [comment]
Hi @dasdebashish17, thanks for your report.

We will take a look for a coming release.

Cheers

**Comment by ccordoba12** [comment]
This should have been fixed in PR #12912, which will be part of our next version (4.1.4), to be released this week. Please update to it when it's out and let us know if that's the case or not.

**Comment by goanpeca** [comment]
Closing due to lack of response.
